shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -2081,6 +2081,11 @@ - (void)removeEnableFlutterTextInputViewAccessibilityTimer {
 }
 
 - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
+  if (_viewController.view == nil) {
```

Comment:
It isn't obvious why we need it here. Can you add a comment? you can also link to the github issue.

Reply:
Done O(∩_∩)O